cparser-scm-1.rockspec

```diff
@@ -0,0 +1,28 @@
+package = 'cparser'
+version = 'scm-1'
+source  = {
+    url    = 'https://github.com/facebookresearch/CParser',
```

Comment:
`https://` is interpreted as for tarball downloading. Use `git+https://` instead.

See `source.url` description here: https://github.com/luarocks/luarocks/wiki/Rockspec-format#build-rules

Reply:
The main version uses 
```
    url    = 'git://github.com/facebookresearch/cparser',
```
which hopefully works already.